src/abstractdatatable/abstractdatatable.jl

```diff
@@ -437,12 +437,13 @@ end
 Indexes of complete cases (rows without null values)
 
 ```julia
-completecases(dt::AbstractDataTable)
+completecases(dt::AbstractDataTable[, indices])
 ```
 
 **Arguments**
 
 * `dt` : the AbstractDataTable
+* `indices` : which columns to check for `Null`s
```

Comment:
Done. In fact I wrote the `cols` version of `nonunique` and the associated docstring, so I should have been in a position to get it right the first time around...